### PR TITLE
Convert Subnet in  1 command

### PR DESCRIPTION
**Pull request by sukantoraymond**
## Why this should be merged
Creates a SOV blockchain in 1 command (subnet deploy)

## How this works

Call node create (use devnet with latest avalanchego etna tag)
Call subnet create
Call subnet deploy 

## How this was tested
subnet deploy works

## How is this documented
NA

- - -
This change is [Reviewable](https://reviewable.io/reviews/ava-labs/avalanche-cli/2230)

### CODE REVIEW

**Comment by meaghanfitzgerald** [comment]
*[`cmd/blockchaincmd/deploy.go` line 730 at r1](https://reviewable.io/reviews/ava-labs/avalanche-cli/2230#gh-1792650729:-O8mP4EY2W4bzOtcHmiO:b-vvh5s8) ([raw file](https://github.com/ava-labs/avalanche-cli/blob/77c0fae294d08679903e661888857e19c44ccef1/cmd/blockchaincmd/deploy.go#L730)):*
<details><summary><i>Previously, sukantoraymond wrote…</i></summary><blockquote>

No longer asking which private key to use to pay because I think we should jsut make the main funded account default
</blockquote></details>

for this initially, yes, but we should probably include this as an option to users after the rest of the flow is ironed out

**Comment by meaghanfitzgerald** [comment]
*[`cmd/blockchaincmd/deploy.go` line 739 at r1](https://reviewable.io/reviews/ava-labs/avalanche-cli/2230#-O8mPWO9CmixyEfW2ahz:-O8mPWO9CmixyEfW2ai-:undefined) ([raw file](https://github.com/ava-labs/avalanche-cli/blob/77c0fae294d08679903e661888857e19c44ccef1/cmd/blockchaincmd/deploy.go#L739)):*
> ```Go
> 	} else {
> 		if err := app.UpdateSidecarNetworks(&sidecar, network, subnetID, blockchainID, "", "", nil); err != nil {
> 			return err
> ```





_Suggestion:_
```Go
		ux.Logger.GreenCheckmarkToUser("L1 is successfully converted to sovereign")
	} else {
		if err := app.UpdateSidecarNetworks(&sidecar, network, subnetID, blockchainID, "", "", nil); err != nil {
			return err
```

File: cmd/blockchaincmd/deploy.go [hunk 1]
```diff
@@ -684,62 +686,59 @@ func deployBlockchain(cmd *cobra.Command, args []string) error {
...
+			genesisPrivateKey,
+			common.HexToAddress(sidecar.PoAValidatorManagerOwner),
+			avaGoBootstrapValidators,
+		); err != nil {
+			return err
+		}
+		ux.Logger.GreenCheckmarkToUser("Subnet is successfully converted into Subnet Only Validator")
+	} else {
+		if err := app.UpdateSidecarNetworks(&sidecar, network, subnetID, blockchainID, "", "", nil); err != nil {
+			return err
```
Comment (meaghanfitzgerald) [inline cmd/blockchaincmd/deploy.go:739]: I used reviewable.io which posted the comments above. Suggestion should be: 
```suggestion
		ux.Logger.GreenCheckmarkToUser("L1 is successfully converted to sovereign")
	} else {
		if err := app.UpdateSidecarNetworks(&sidecar, network, subnetID, blockchainID, "", "", nil); err != nil {
			return err
```

File: cmd/blockchaincmd/helpers.go [hunk 2]
```diff
@@ -93,3 +93,21 @@ func UpdateKeychainWithSubnetControlKeys(
 	}
 	return nil
 }
+
+func getClusterNameFromList() (string, error) {
```
Comment (arturrez) [inline cmd/blockchaincmd/helpers.go:97]: nit: move it to `pgk/node` as everything else in the refactor PR

**Review by arturrez — APPROVED** [review]
with 1 nit

**Pull request merged**